crates/apub/src/activities/block/mod.rs

```diff
@@ -107,13 +107,57 @@ impl Object for SiteOrCommunity {
   }
 }
 
+impl Actor for SiteOrCommunity {
```

Comment:
I also implemented `Either` for Actor.

Reply:
I published those changes as 0.6.4 now.

https://github.com/LemmyNet/activitypub-federation-rust/releases/tag/0.6.4